examples/geomap/README.md

```diff
@@ -12,8 +16,8 @@
 ## Technologies
 
 - Pigeon Map: https://pigeon-maps.js.org/docs/installation
+- crystal-lang: https://crystal-lang.org
 
 ## Getting Started
 
 @TODO
```

Comment:
I am going to add the whole description of the project by the end of the week, then I'll create the PR.